internal/pkg/template/templates/workloads/partials/cf/eventrule.yml

```
@@ -3,8 +3,13 @@ Rule:
     'aws:copilot:description': "A CloudWatch event rule to trigger the job's state machine"
   Type: AWS::Events::Rule
   Properties:
+    {{if eq .ScheduleExpression "none"}}
+    ScheduleExpression: "rate(5 minutes)"
+    State: DISABLED 
+    {{ else }}
```

Tiny go template changes to remove extra new lines 🙏 
```suggestion
    {{- if eq .ScheduleExpression "none"}}
    ScheduleExpression: "rate(5 minutes)"
    State: DISABLED 
    {{- else }}
```